src/GCRealTimeMon/CallStackResolution/CallStackManager.cs

```diff
@@ -0,0 +1,122 @@
+﻿using Microsoft.Diagnostics.Symbols;
+using Microsoft.Diagnostics.Tracing;
+using Microsoft.Diagnostics.Tracing.Etlx;
+using Microsoft.Diagnostics.Tracing.Parsers;
+using Microsoft.Diagnostics.Tracing.Parsers.Clr;
+using Microsoft.Diagnostics.Tracing.Session;
+using realmon.Utilities;
+using System;
+using System.IO;
+
+namespace realmon.CallStackResolution
+{
+    public static class CallStackManager
+    {
+        private static SymbolReader m_symbolReader;
+
+        public static TraceLogEventSource InitializeAndRegisterCallStacks(Configuration.Configuration configuration, 
+                                                                          TraceEventSession traceEventSession, 
+                                                                          int processId)
+        {
+            // TODO: Add configurations for the Symbol Reader Text Output path and the NT_SymbolPath.
+            m_symbolReader = new SymbolReader(TextWriter.Null, SymbolPath.SymbolPathFromEnvironment); 
```

Comment:
>SymbolPath.SymbolPathFromEnvironment [](http://example.com/codeflow?start=63&length=36)

I wonder if we should populate the symbol path with what you pointed out in the readme if the user's _NT_SYMBOL_PATH is not specified?